airflow_dbt_python/hooks/remote/__init__.py

```diff
@@ -33,7 +33,7 @@ class DbtRemoteHook(ABC, LoggingMixin):
     """
 
     @abstractmethod
-    def download(
+    def _download(
```

Comment:
Any reason for adding an underscore prefix to all these methods? I know some linters could report this as a private method when calling it, but beyond that nothing is really enforcing the privacy.

Reply:
These methods started to overlap with methods in GCSHook. 
So I had a choice: rename them or add a prefix. I chose the latter since they are not used as public.